packages/edge-login-ui-rn/src/common/locales/strings/enUS.json

```diff
@@ -115,5 +115,7 @@
   "landing_already_have_account": "Already have an account? Sign in",
   "pin_not_enabled": "PIN is not enabled for this account",
   "invalid_pin": "Invalid PIN",
-  "account_locked_for": "Account locked for \n%1$s more seconds"
+  "account_locked_for": "Account locked for \n%1$s more seconds",
+  "otp_modal_reset_headline": "2FA Reset Requested",
+  "otp_modal_reset_body": "2FA disable has been requested by another device for the following accounts on this device:\n"
```

Comment:
This needs a bit more wording. Please change to:

"URGENT: 2FA reset has been requested by another device for the following accounts **usernames separated by commas**.\n\n

If you did not request a 2FA reset, please go to Settings -> 2 Factor and click Keep 2FA. Then change your password."

I assume that the user will still get a 2FA reset warning inside the Settings -> 2FA scene when they enter that.